packages/jest-snapshot/src/__tests__/utils.test.js

```diff
@@ -195,12 +195,57 @@ test('serialize handles \\r\\n', () => {
 
 describe('DeepMerge', () => {
   it('Correctly merges objects with property matchers', () => {
-    const target = {data: {bar: 'bar', foo: 'foo'}};
+    /* eslint-disable sort-keys */
+    // to keep keys in numerical order rather than alphabetical
```

Comment:
Are you just adding a missing test or is this a changed behavior?

Reply:
This is adding a missing test for the case of nested matchers.